cmd/auth/command.go

```diff
@@ -26,7 +26,7 @@ import (
 // Command is the auth subcommand
 var Command = &cobra.Command{
 	Use:     "auth",
-	Short:   "Manages the platform auth",
+	Short:   "Manages authentication",
```

Comment:
Maybe add `settings` at the end?
```suggestion
	Short:   "Manages authentication settings",
```